src/Drive.php

```diff
@@ -290,7 +290,25 @@ public function getResources(string $path = "/"): array
         $resources = [];
         $webDavClient = $this->createWebDavClient();
         try {
-            $responses = $webDavClient->propFind(rawurlencode(ltrim($path, "/")), [], 1);
+            $properties = [
```

Comment:
Can we construct these property strings by using the names in the ResourceMetadata enum?

Reply:
done